composables/album.ts

```diff
@@ -8,12 +9,24 @@ export function useAlbum(options: UseAlbumOptions) {
   const { id } = options;
 
   return reactiveApi(
-    useLazyAsyncData(`album-${id}`, () => new AlbumApi().albumIdGet({ id })),
+    useLazyAsyncData<AlbumModel>(
```

Comment:
I would call this a very wordy solution, since it adds multiple lines of code in a lot of places. We should chat if this is the best solution.